modules/ROOT/pages/_partials/app-store-script.adoc

```diff
@@ -0,0 +1,15 @@
+=== Submission to the App Store
+
+The following does not apply if you have installed Couchbase Lite with Carthage or Cocoapods. With those package managers, Couchbase Lite is built with the application itself according to the architecture type of the target (emulator or device).
+
+The Couchbase Lite framework available on the link:https://couchbase.com/downloads[downloads page] contains a build for both the simulator (x86_64) and iOS devices (ARM). Submissions to the App Store of a binary that contains the simulator architecture is not supported therefor you must remove it from **CouchbaseLite.framework** before App Store submission. The following steps describe how to set up a build phase in Xcode to do this automatically.
```

Comment:
Just occurred to me that we may have an issue even w/ Cocoapods since with Couchbase Lite EE version that's distributed via binaries only.  Workarounds specified here  may not quite work  since they update settings to be arm only ...https://github.com/CocoaPods/CocoaPods/issues/7111 . (@pasin )